src/interfaces/idleUser.type.ts

```diff
@@ -0,0 +1,21 @@
+export type idleUserArray = {
```

Comment:
Can the name be changed to match the use case of this type in `idealUsers/section/index.tsx` if this is not being used anywhere else? 

Because the word `idleUserArray`  I would assume means something like this :
`
    type idealUserArray = idealUser[]
`

#### Nitpick: It is a general practice to name types where the first letter is Capital.

Reply:
Making those editions
BTW I think it is when we declare interfaces, we need them to be PascalCased but for types, it can start with small letter

Reply:
I feel maintaining consistency in the way types are declared (regardless of their type interface/types) makes it easier to perform changes.

Reply:
understood. Will make the changes